lib/mailersend/client.rb

```diff
@@ -1,20 +1,17 @@
 # frozen_string_literal: true
 
 require 'http'
-require 'dotenv/load'
 
 API_URL = 'https://api.mailersend.com/v1'
 API_BASE_HOST = 'api.mailersend.com'
 
-Dotenv.require_keys('MAILERSEND_API_TOKEN')
-
 # mailersend-ruby is a gem that integrates all endpoints from MailerSend API
 module Mailersend
   attr_reader :api_token
 
   # Inits the client.
   class Client
-    def initialize(api_token = ENV['MAILERSEND_API_TOKEN'])
+    def initialize(api_token: Rails.configuration.mailersend_api_token)
```

Comment:
Hi! This changes the type of the parameter, from positional argument to a keyword argument. Also note that right now this is a ruby library, it not depends on rails and can be used without it.

Reply:
@rbngzlv correct. I only opened this PR as a draft since several people requested getting rid of dotenv, so it is more of a solution preview not a PR that I intend to have merged.

For Rails it would be better to have a separate repo that also defines mailersend as an `ActionMailer::DeliveryMethod` so you can configure it to deliver mail instead of smtp like that ```config.action_mailer.delivery_method = :mailersend```